xmlserializer/XmlStreamDocSink.cpp

```diff
@@ -54,7 +52,7 @@ bool CXmlStringDocSink::doProcess(CXmlDocSource& xmlDocSource,
         return false;
     }
 
-    _strResult.append((const char*)pcDumpedDoc);
+    _output << (const char*)pcDumpedDoc;
```

Comment:
nice exmaple of the EVIL of hungarian: this is NOT a pcXXX this is a xmlChar... and despite the guess work, it is not a char *....